net/gallery-dl/Portfile

```diff
@@ -4,17 +4,17 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        mikf gallery-dl 1.25.5 v
+github.setup        mikf gallery-dl 1.25.6 v
 github.tarball_from releases
 distname            gallery_dl-${github.version}
 
 categories          net
 maintainers         {@akierig fastmail.de:akierig} openmaintainer
-revision            0
+revision            1
```

Comment:
"revision" field should be reset to 0 when updating the version

Reply:
thanks, I was debating this because I added in the man page part. Makes more sense.